hashes/extended_tests/schemars/README.md

```diff
@@ -4,10 +4,6 @@ Run as usual with `cargo test`.
 
 ## Minimum Supported Rust Version (MSRV)
 
-To run the tests with the MSRV you will need to pin `serde`:
-
-```bash
-cargo update -p serde --precise 1.0.156
-cargo update -p regex --precise 1.7.3
-cargo update -p chrono --precise 0.4.24
-```
+To run the tests with the current MSRV you will need to pin some
+dependencies. See the `hashes/contrib/extra_tests.sh` script for the
+current list of pins.
```

Comment:
If we want to reference that maybe we should have the list in some easier format or have the pinning script separated so that users can run it. Oh, actually this may be a great idea in general that works around the current limitations of cargo resolver!